The Big Guide to Web Components Built With React/The Big Guide to Web Components Built With React.md

```diff
@@ -0,0 +1,434 @@
+# The Big Guide to Web Components Built With React
+
+Our team at WTW wanted to find a way to deploy multiple small applications to the same webpage in a way that allows greater than a dozen teams to separately develop, build, and deploy their apps in isolation, and without having to have multiple gates (other people, teams, etc) in order to get new code into production. (This approach is often called [Micro Frontends](https://micro-frontends.org/).)
+
+We researched a lot of different way of handling this problem, but finally landed on utilizing Web Components as our solution. This allows us to deploy individual, isolated Javascript applications together on a single page without much responsibility on other teams to expend a lot of work to integrate those components.
+
+The primary web technology our shop uses is React, so using React with our Web Components decision was a primary need for us. We've learned a lot as we've treaded the path of building isolated Web Components with React, and this article will detail much of that learning.
```

Comment:
I'm having a hard time parsing "using React with our Web Components decision".  This sentence also mixes present and past tense.  Perhaps "so we would like to build our Web Components with React".  

The past tense of "tread" is "tread".

Reply:
My spellchecker failed me. 😧